generator-eui/component/templates/test.tsx

```diff
@@ -14,11 +14,10 @@ import { <%= componentName %> } from './<%= fileName %>';
 
 describe('<%= componentName %>', () => {
   test('is rendered', () => {
```

Comment:
missed this in the original changes
```suggestion
  it('renders', () => {
```